### PR TITLE
Add start and shutdown HANA instance commands to CLI.

### PR DESCRIPTION
**Pull request by vipulha**
Added start and shutdown commands to CLI.
Note that the SDK is already updated in previous PR: https://github.com/Azure/azure-hanaonazure-cli-extension/pull/25

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -79,6 +79,24 @@ az hanainstance monitor enable \
     --hana-db-password $HANA_DB_PW
 ```
 
+To restart a specific HANA instance:
+
+```
+az hanainstance restart --resource-group $RESOURCE_GROUP --instance-name $HANA_INSTANCE_NAME
```
Comment (lagalbra) [inline README.md]: Open a separate PR to correct the fact that this wasn't already in the README

**Review by PakDLiu — APPROVED** [review]

**Review by lagalbra — CHANGES_REQUESTED** [review]
Did you test building the CLI with these changes and starting/stopping a HanaInstance?

File: README.md (hunk 1 above)
Comment (vipulha) [reply README.md]: Removed the restart part from README.

**Comment by vipulha** [comment]
The changes have been unit tested by building the CLI & verifying that it generates the correct REST API request.

**Review by lagalbra — APPROVED** [review]

**Review by jovillas — APPROVED** [review]

**Pull request merged**